g1_map_robot.cydsn/ZumoLibrary/Motor.c

```diff
@@ -57,6 +63,117 @@ motor_forward (uint8 speed, uint32 delay)
   SetMotors (0, 0, speed, speed, delay);
 }
 
+/**
+ * @brief Moving motors forward for amount of seconds
+ * @details This function calls for balancing algorithm internally
+ * @param speed : speed value
+ * @param seconds : time in seconds
+ */
+#if 0
```

Comment:
Why there are two different variants? Maybe delete one of them or comment them out as the reason there is two.